pkg/agent/dispatcher.go

```diff
@@ -46,18 +45,10 @@ type Dispatcher struct {
 	// message and will send it down the SignalSender so the original sender can use it.
 	waiters map[uint64]util.SignalSender[*MonitorResult]
 
-	// A message is sent along this channel when an upscale is requested.
-	// When the informant.NewState is called, a goroutine will be spawned that
-	// just tries to receive off the channel and then request the upscale.
-	// A different way to do this would be to keep a backpointer to the parent
-	// `State` and then just call a method on it when an upscale is requested.
-	// However, we do not do this because it causes a reference cycle, which
-	// interferes with go's ability to run finalizers - which nhooyr/websocket
-	// uses to manage connections. Thus, if a panic were to happen, the connection
-	// might not get dropped.
-	notifier util.CondChannelSender
-
-	// nextTransactionID represents the current transaction id. When we need a new one
```

Comment:
Unnecessary comment change on this line? ("nextTransactionID" -> "This counter")

Reply:
I think that was a rebase/merge error on my part.